src/compute-client/src/controller/orchestrator.rs

```diff
@@ -52,20 +50,17 @@ impl ComputeOrchestrator {
         instance_id: ComputeInstanceId,
         replica_id: ReplicaId,
         location: ComputeReplicaLocation,
-    ) -> Result<(Vec<String>, TimelyConfig), anyhow::Error> {
+    ) -> Result<(Vec<String>, usize, Vec<String>), anyhow::Error> {
```

Comment:
Can you also add a doc to this function explaining the meaning of the return type?

Reply:
I was lazy, thanks for calling me out!